ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml

```diff
@@ -143,6 +143,22 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- always_run: false
+  as: e2e-aws-gatewayapi-conformance
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
The PR https://github.com/openshift/api/pull/2081 already got merged, so update env `FEATURE_SET` to `TechPreviewNoUpgrade` accordingly.